vita3k/host/dialog/include/host/dialog/filesystem.hpp

```diff
@@ -105,8 +103,4 @@ Result pick_folder(std::filesystem::path &resulting_path, const std::filesystem:
  */
 std::string get_error();
 
-} // namespace filesystem
-
-} // namespace dialog
-
-} // namespace host
+} // namespace host::dialog::filesystem
```

Comment:
Please keep newlines at the end of files

Reply:
Fixed! For all project files

I tried to add [InsertNewlineAtEOF](https://releases.llvm.org/16.0.0/tools/clang/docs/ClangFormatStyleOptions.html#insertnewlineateof) in the .clang-format file to enforce this rule but it is only supported by clang-format 16+ (CI runner ubuntu-22.04 has version 15)